beefy-node/node/src/service.rs

```diff
@@ -1,18 +1,16 @@
 //! Service and ServiceFactory implementation. Specialized wrapper over substrate service.
-use std::sync::Arc;
-use std::time::Duration;
-
-use sc_client_api::{ExecutorProvider, RemoteBackend};
-use sc_consensus_aura::{ImportQueueParams, SlotProportion, StartAuraParams};
-use sc_executor::native_executor_instance;
-use sc_finality_grandpa::SharedVoterState;
-use sc_service::{error::Error as ServiceError, Configuration, TaskManager};
-use sc_telemetry::TelemetrySpan;
-
-use sp_consensus_aura::sr25519::AuthorityPair as AuraPair;
-use sp_inherents::InherentDataProviders;
-
-use beefy_node_runtime::{self, opaque::Block, RuntimeApi};
+use {std::sync::Arc, std::time::Duration};
```

Comment:
Can we use rustfmt's `--check --config merge_imports=true` to have the formatter enforce uniform imports?

See [example here](https://github.com/tomusdrw/rust-web3/blob/master/.github/workflows/lint.yml#L35).